src/service/gateway.proto

```diff
@@ -17,6 +20,18 @@ enum close_state {
 
 /* general */
 
+message empty {}
+
+message gateway_success_resp {
+  bytes resp = 1;
```

Comment:
can we make success_resp be  empty? including actual data seems to imply we're missing some actual types for the responses that have them. 
i.e. make 

```
message gateway_success_resp {}
```
and remove the `empty{}`

Reply:
That is on my to-do list but was leaving it til later..IE before the rust client comes along....what is there now works fine for Erlang client

Reply:
yeah just that once validators are deployed with this we'll have issues removing fields there

Reply:
Especially since we're already adjusting the protos again?